examples/platform/linux/AppMain.cpp

```diff
@@ -492,6 +497,21 @@ int ChipLinuxAppInit(int argc, char * const argv[], OptionSet * customOptions,
     return 0;
 }
 
+namespace {
+static constexpr size_t kFabricId = 1;
+
+ExampleOperationalCredentialsIssuer gOpCredsIssuer(kFabricId);
+PersistentStorage gStorage;
+
+CHIP_ERROR PrepareJointFabricCluster()
+{
+    SetPersistentStorageDelegate(&gStorage);
+    SetOperationalCredentialsIssuer(&gOpCredsIssuer);
+    SetChipToolKvs(LinuxDeviceOptions::GetInstance().chipToolKvs);
+    return CHIP_NO_ERROR;
+}
+} // namespace
```

Comment:
Linux apps all share this. Having these global setters and fixed fabric ID breaks other assumptions of the Linux examples